### ISSUE TITLE
I'm not able to find the data folder.

### ISSUE THREAD

**Issue by iMac-ant** [Denbergvanthijs/imbDRL#4]
Dear @Denbergvanthijs I tried to run the scripts i ReadMe in Getting Started, but seems that there are not the data files, e.g. 'credit0.csv'. Could you provide them?

Thanks in advance,

Antonio

**Comment by Denbergvanthijs** [comment]
Hi Antonio,

The creditcard data can be downloaded from the Kaggle link in the readme. After downloading, the data needs to be preprocessed with the correct functions.

Kind regards,
Thijs

**Comment by iMac-ant** [comment]
Okay thank you!

I'll close this issue. 

P.S. 
Remember to replace sklearn with scikit-learn inside the requirements file to correctly install the library.

Thank you for providing this algorithm. I'll use it in a mine CyberSec research.

**Comment by Denbergvanthijs** [comment]
Sounds like an awesome project, good luck! Ill change the requirements right away

**Comment by iMac-ant** [comment]
Hi @Denbergvanthijs, could you suggest me an easy way to log the reward value for each step?
